polyphemus/lib/client/jsx/etl/redcap-form.tsx

```diff
@@ -289,6 +289,12 @@ const RedcapAttribute = ({att_name, attribute_value, update}:{
 
   let valueComponent;
 
+  const handleUpdateValue = useCallback((field_name: string, newValue: any) => {
```

Comment:
I was thinking that, and this doesn't have to fit into this PR, we might want to consider creating some custom hooks that handle some more of these cases where we're essentially wrapping existing hooks with a bit of very generic logic.

Something like, `useStateObjectWithFilter` ?  Especially since callbacks around other hooks gets kinda verbose doing over and over again.

Reply:
Ah, hm, let me see if I can wrap my head around what that would look like. I think I see what you mean, that these are kind of all thin wrappers around `update()` (which, at the end, is a `setState` type call).